File: pywebsocket/MyService/Message.py
from __future__ import annotations
from MyService.SubService.SubMessage import SubMessage as ImportedRef__MyService_SubService_SubMessage
from typing import TypedDict

class Message(TypedDict):
    def __init__(self):
        super().__init__()
        self.__reflection = 'MyService.Message'

    __reflection: str
    name: str
    value: int
    tested: bool
    sub: ImportedRef__MyService_SubService_SubMessage[Message]

File: jswebsocket/MyService/Message.ts
import {SubMessage as $MyService_SubService_SubMessage} from "./SubService/SubMessage";
export class Message
{
    __reflection: string = "MyService.Message";
    public name: string;
    public value: number;
    public tested: boolean;
    public sub: $MyService_SubService_SubMessage<Message>;
}